qutebrowser/mainwindow/statusbar/backforward.py

```diff
@@ -23,6 +23,9 @@
 
 
 class Backforward(textbase.TextBase):
+    def __init__(self):
+        super().__init__()
+        self.enabled = False
```

Comment:
This should be after the docstring, and take a `parent=None` argument which it passes to `super().__init__()`. Haven't looked at the rest yet, but I'll do so later and fix this up, depending on who's faster :wink:

Reply:
Ok, fixed.